backend/src/main/java/de/robinschatzl/gameapp/backend/security/UserService.java

```diff
@@ -1,17 +1,20 @@
 package de.robinschatzl.gameapp.backend.security;
 
-import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import java.util.NoSuchElementException;
 
 @Service
-@RequiredArgsConstructor
```

Comment:
why do you remove the RequiredArgsConstructor but added it in another class?
You should use it consistently.

Reply:
it was just a test wasn't chage it back thx for that one